unified_targets/docs/Manufacturers.md

```diff
@@ -34,6 +34,7 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |SPRO|Seriously Pro Racing (SP Racing)|http://seriouslypro.com/|
 |TACO|Taco RC|https://taco-rc.com/|
 |TCMM|Shenzhen Xinmumu Science and Technology Co (TCMM)|http://www.tcmm-rc.cn/|
+|TLON|TALON|https://www.heli-nation.com/|
```

Comment:
This still needs to be removed.

Reply:
@mikeller changed. sorry for the mistake.